src/scripts/main.js

```diff
@@ -1,3 +1,91 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('tbody');
+const addRowButton = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const minRows = 2;
+const maxRows = 10;
+const minColumns = 2;
+const maxColumns = 10;
```

Comment:
Don't need to duplicate variables with the same values, it's a bad practice
```suggestion
const minValue = 2;
const maxValue = 10;
```